google-cloud-storage/src/main/resources/reference.conf

```diff
@@ -0,0 +1,6 @@
+alpakka.googlecloud.storage {
```

Comment:
Other Google Cloud connector (Pub/Sub) has used `alpakka.google.cloud.pubsub` namespace. So this one should follow suit.